### ISSUE TITLE
Implement access.log and error.log in standard web server format

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/GlobaLeaks#2420]
This ticket is to collect detail about the implementation of access.log and error.log in standard web server format.

Currently this is implemented as part of the globaleaks.log file where debug and access information is mixed up.

It has been considered valuable to perform the following changes:
- The access and error information should be split out the file globaleaks.log
- The IP address and User agent of users could be logged for all authenticated user actions with the obvious exception of the whistleblower users.

This changes are considered in order to make it possible to identify fraud activities and implement proper accounting for privileged users.

**Comment by fpietrosanti** [comment]
For all security sensitive actions, such as authentication success/failure (and further to come, such as encryption failure, DOS detect, etc), i would suggest to split a dedicated security.log so that access.log and error.log are purely HTTP-related log in apache format, splitted out from globaleaks.log ?

**Comment by evilaliv3** [comment]
Definitely yes, but this aspect is pretty out of topic for this ticket.

This ticket is about to implement a standard access.log following combined log format.

Please keep tickets closely-knit, clean and in topic and when the suggestion is out of the topic consider opening new tickets.

**Comment by fpietrosanti** [comment]
@evilaliv3 yes, but standard apache-alike access log doesn't track authentication parameters of users, so are useless for security auditing purposes
